modules/qna/src/views/full/utils/qnaList.utils.ts

```diff
@@ -218,14 +220,15 @@ export const fetchReducer = (state: State, action): State => {
       expandedItems: { ...state.expandedItems, [id]: true }
     }
   } else if (action.type === 'deleteQnA') {
-    const { index, bp } = action.data
+    const { index, bp, getQnaCountByTopic } = action.data
     const newItems = state.items
 
     if (index === 'highlighted') {
       bp.axios
         .post(`/mod/qna/questions/${state.highlighted.id}/delete`)
         .then(() => {})
         .catch(() => {})
+      getQnaCountByTopic?.()
```

Comment:
didn't know that feature `?.()` is this ts sugar syntax for

`!!getQnaCountByTopic && getQnaCountByTopic()`

Reply:
yep